### ISSUE TITLE
[BITV] 9.1.4.3/3.2 - The input field "Collaborative tags" - The placeholder text (placeholder attribute) does not meet the minimum color contrast ratio. (2)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36952]
- [x] https://github.com/nextcloud/server/pull/37065

![5e2cbbfe0fd4258eab94ad65371fb1fa](https://user-images.githubusercontent.com/1315170/222223274-ac4789e0-6d74-44b5-a8ba-f736a623faa1.png)

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-6fe46b78da-v3-n2

</details>

- [x] 9.1.4.3/10.2 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-6fe46b78da-v10-n2
